src/__configuration__/resources/index.tsx

```diff
@@ -141,14 +141,6 @@ export const resources: ResourceBucket[] = [
                 bugLabels: ['ui'],
                 workSpace: 'ui',
```

Comment:
you may need to remove the workspace attribute here too. We no longer have a `ui` folder in that repository. `bugLabels` is no longer applicable